### ISSUE TITLE
Hledger 1.30 fails to install with stack

### ISSUE THREAD

**Issue by aragaer** [simonmichael/hledger#2042]
```
hledger                          > /tmp/stack-d2188e33b35c6e2b/hledger-1.30/Hledger/Cli/Commands/Demo.hs:63:3: error:
hledger                          >     • Exception when trying to run compile-time code:
hledger                          >         /tmp/stack-d2188e33b35c6e2b/hledger-1.30/embeddedfiles/add.cast: openFile: does not exist (No such file or directory)
hledger                          >       Code: (embedFileRelative "embeddedfiles/add.cast")
hledger                          >     • In the untyped splice:
hledger                          >         $(embedFileRelative "embeddedfiles/add.cast")
hledger                          >    |
hledger                          > 63 |   $(embedFileRelative "embeddedfiles/add.cast"),      -- https://asciinema.org/a/567935 The easiest way to start a journal (add)
hledger                          >    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Using stack-2.9.3, ghc-9.6.1

**Comment by glguy** [comment]
Some files were missing in the extra-source-files section: https://github.com/glguy/hledger/commit/a9f03fa4a8d1e476c38225fa6755e531383628ce

**Comment by simonmichael** [comment]
Thank you for the report, I have released hledger-1.30.1 on Hackage and updated the Install page.

Dupe: #2043
